### PR TITLE
Update py-sqlparse port.

### PR DESCRIPTION
**Pull request by xeron**
* new version 0.2.1
* add myself as maintainer
* remove Id line
* use python portgroup variables
* use pypi livecheck
* indentation cleanup and small reorder

Latest version is 0.2.2 but I'm using 0.2.1 for now because `mycli` depends on `<=0.2.1`.

### CODE REVIEW

File: python/py-sqlparse/Portfile [hunk 1]
```diff
@@ -1,33 +1,30 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```
Comment (xeron) [inline python/py-sqlparse/Portfile]: Fixed.

**Comment by xeron** [comment]
@mojca I've updated commit message and enabled `edits from maintainers`. Still not sure why you need it, github allows to merge PRs which have no conflicts no matter of rebasing on the latest master.

So far macports is the first project where I see requirement to allow edits to PRs from maintainers.

**Comment by larryv** [comment]
@xeron If a maintainer wants to make minor modifications before merging/rebasing, they have to be able to push to your PR branch, or the GitHub pull request page will not recognize the merge.

**Comment by xeron** [comment]
@larryv what if PR doesn't require any further changes? Rebase on the latest master is not required for merge until you have conflicts.

**Comment by larryv** [comment]
@xeron You're correct; I assumed that @mojca wanted to make some more changes. But we will probably encourage that contributors open all PRs this way so that we don't have to ask them to make minor changes.

@mojca The push access is only required when you want to change the PR branch before rebasing. If you don't want to make any changes, then using the "Rebase and merge" button works fine.

**Comment by mojca** [comment]
So basically clicking "rebase and merge" will work, but using command-line won't unless the changes are rebased and force-pushed?

**Comment by mojca** [comment]
@xeron: sorry, we are all learning :(

**Comment by larryv** [comment]
We should continue this discussion in the "Properly merging pull requests" thread on macports-dev, but the general idea is that if you—the committer—want to make changes before closing the PR, you have to push those changes to the contributor's PR branch first. If you have no changes to make, you don't have to touch the contributor's PR branch.

**Pull request merged**